packages/design-system/src/components/SearchBar/SearchBar.md

```diff
@@ -0,0 +1,46 @@
+### SearchBar
+
+
+### Compact search bar
+
+Used for mobile devices or smaller resolution designs
+
+```js
+const onChange = value => {
+  console.log('Search term: ', value);
+}
+<SearchBar size="compact" onChange={onChange} />
+```
+
+### Search bar with onChange handler
+
+Most common use case for search bar is with onChange handler. It can be enhanced with debounce method to reduce many callbacks
+
+```js
+const onChange = value => {
```

Comment:
can you make the loader visible in docs? maybe some timeout to mock a call to api